openc3/python/openc3/interfaces/file_interface.py

```diff
@@ -102,6 +100,7 @@ def connect(self):
             self.listener.schedule(event_handler, self.telemetry_read_folder, recursive=self.recursive)
             self.listener.start()
 
+        super().connect()
```

Comment:
Sending the command in the `connect` method only works if we do the `super()` last.